docs/features/proto-generation.rst

```diff
@@ -2,17 +2,18 @@
 
 Proto generation
 ================
+
 To be able to generate proto files you need to register your service first.
 To do so please refer to :ref:`getting started <getting_started>` at section service registration
 
 Description
 -----------
-Proto files contain the classes, descriptors and controller logic (``pb2.py`` files) and proto message syntax (``.proto`` file) necessary to run a grpc server.
-In django-socio-grpc, proto files are generated from a ``grpc_action`` request / response contents (see :ref:`grpc action <grpc_action>`).
 
+Proto files contain the classes, descriptors and controller logic (``pb2.py`` files) and proto message syntax (``.proto`` file) necessary to run a grpc server.
 
-You can also generate proto files from a service that has a ``serializer_class`` attribute (mapped to a ``ProtoSerializer`` class) and inherits from a mixin that automatically generates grpc_actions
+In django-socio-grpc, proto files are generated from  from a service that has a ``serializer_class`` attribute (mapped to a ``ProtoSerializer`` class) and inherits from a mixin that automatically generates grpc_actions (see see :ref:`Generic Mixins <Generic Mixins>`)
 
+You can also generate proto files from ``grpc_action`` request / response contents (see :ref:`grpc action <grpc_action>`).
```

Comment:
Original is more correct, generation comes from grpcaction, serializers are converted to a grpc action and not mandatory